pkg/api/azenvtypes.go

```diff
@@ -148,6 +148,30 @@ var (
 		ImageVersion:   "2019.09.24",
 	}
 
+	// AKSWindows2019OSImageConfig is the AKS image based on Windows Server 2019
+	AKSWindowsServer2019OSImageConfig = AzureOSImageConfig{
+		ImageOffer: "aks-windows",
+		ImageSku: "2019-datacenter-core-smalldisk",
+		ImagePublisher: "microsoft-aks",
+		ImageVersion: "17763.737.190923",
+	}
+
+	// AzureStackWindowsServer2019OSImageConfig is the 'vanilla' Windows Server 2019 image to be used to with Azure Stack deployments
+	AzureStackWindowsServer2019OSImageConfig = AzureOSImageConfig{
```

Comment:
@rjaini as FYI - this change moves where the default imaging values for AzureStack gets defined (from consts.go to azenvtypes.go)

Reply:
Thanks! I think this fits in better with the existing implementation. (Hope you agree and aren't under the delusion that folks just tell you what to do :/)

Reply:
I'm unhappy with how different image specification works for Linux vs Windows. 
I looked into unifying it but it looked like it was going to be very painful to do so and support existing API models. 
I would like to see converge for V2 tho.